### DISCUSSION TITLE
How does max_concurent_with_start_early work

### DISCUSSION THREAD

**Discussion by paullepa** [ericaltendorf/plotman#292] [Q&A]
Seems like there is two 'max' limits - could someone clarify for me how to apply them?
I'm would like to run up to 10 jobs at a time (disk intensive jobs), but can run 3 more jobs in phase one.
So, I set my settings to max_concurent = 10, max_concurent_with_start_early=13  
but the most I get is 10 jobs?

Thanks,

 max_concurrent: The maximum number of plots to have for this job at any given time.
  # max_concurrent_with_start_early: The maximum number of plots to have for this job at any given time including
  #                                  phases that started early.

**Comment by BasilHorowt** [comment]
"max_concurrent_with_start_early" is not an option in Plotman, but I searched for it and found that it is an option in "Swar" a different tool.  You will have more luck asking over there: https://github.com/swar/Swar-Chia-Plot-Manager/discussions
